templates/minimal/package.yaml

```diff
@@ -14,6 +14,22 @@ dependencies:
   pypi:
     - sema4ai-actions=1.3.8
 
+dev-dependencies:
+  conda-forge:
+    - pytest=8.3.5
+    - types-requests=2.32.0.20250301
+    - ruff=0.11.12
+    - mypy=1.16.0
+
+dev-tasks:
+  test: pytest tests
```

Comment:
For the minimal I don't think we can bring pytests..
Linters and ruff checks yes but pushing the whole shebang right from the most minimal setup feels a bit lot.

Reply:
Humm, I don't know -- for me the minimal thing to have is a project with tests/lint/type-checking, so, I'd keep it...